src/main/java/xyz/vegaone/easytracking/utils/Status.java

```diff
@@ -0,0 +1,8 @@
+package xyz.vegaone.easytracking.utils;
```

Comment:
I like how you extracted the constants into an Enum. 
It's one of those small things that adds clarity to the code.

Reply:
thank you